gateleen-security/README_security.md

```diff
@@ -63,4 +63,45 @@ Any request with a user group containing 'acl-domain-' will match the defined ro
 
 Note the additional keepOriginal flag which defines if the original role which matched the mapper must be kept in the list of roles or not.
 
-Note: The rolemapper object is validated against the gateleen_security_schema_rolemapper json schema
\ No newline at end of file
+Note: The rolemapper object is validated against the gateleen_security_schema_rolemapper json schema
+
+#### Using request headers
+In ACLs you can use wildcard definitions in the urls. These wildcards can contain request header names and will be replaced with the actual
+values from a request.
+
+The wildcards have to be in the format 
+```
+<header name>
```

Comment:
What was the reason not to use { } which looks more "variablish"?

Reply:
I tried to use `{}` first but got massive problems with the Java Pattern class because normally in a regex pattern the `{}` is reserved for quantifiers like `{5}` or `{3,5}`. I could fix this by adding the `Pattern.LITERAL` option during pattern parsing. However, urls having such quantifiers did not work anymore. Example: `/resources/userspecific/{x-user}/items/([0-9]{6})/foo`. Switching to `<>` solved these problems

Reply:
Ah yes, forgot the quantifier.
Makes sense. < > is something we should not see in a normal URL indeed.

Reply:
Be aware of [Named Capture Groups](https://riptutorial.com/regex/example/2479/named-capture-groups). Nevertheless I know no place where we would use them in URLs ;)